apps/settings/js/media_storage.js

```diff
@@ -608,7 +608,7 @@ require([
 
     handleEvent: function ms_handleEvent(evt) {
       switch (evt.type) {
-        case 'localized':
+        case 'DOMLocalized':
```

Comment:
There isn't a `DOMLocalized` event anymore, only `DOMRetranslated'.  Do you need to run this code also on app's first translation?